scripts/packetBuilder.lua

```diff
@@ -10,6 +12,9 @@ packetBuilder.AddPlayerInventoryItemChange = function(pid, item)
     tes3mp.AddItemChange(pid, item.refId, item.count, item.charge, item.enchantmentCharge, item.soul)
 end
 
+---@param pid integer
+---@param spellsActive table<string, SpellInstance[]>
+---@param action integer
```

Comment:
Can you add an alias for this type? 

I just found out there exists a table with these enumerations: `enumerations.spellbook.ADD`. You could annotate these like this: https://github.com/sumneko/lua-language-server/wiki/Annotations#enum. Please think about this before resolving my other comments where I suggested using `---@alias`. I think it would be much more elegant to use `---@enum` for constants that already exist in the code.